core/src/main/java/tech/tablesaw/api/CategoryColumn.java

```diff
@@ -291,7 +294,7 @@ public int countUnique() {
         return bottom;
     }
 
-    public void add(String stringValue) {
+    public final void add(String stringValue) {
```

Comment:
I'm not sure we need to make this `final`

Reply:
An option would be to move the content of the public method to another with private access modifier. But calling a overideable method from the constructor is considered to be a bad practice. https://stackoverflow.com/questions/3404301/whats-wrong-with-overridable-method-calls-in-constructors

Reply:
I'd probably prefer your second option so that users can still override this method if they have a need to